.github/workflows/checks.yml

```diff
@@ -18,6 +18,7 @@ jobs:
         - python-version: 3.8
           env:
             TOXENV: pylint
+            TOX_PIP_VERSION: 20.3.3
```

Comment:
Why is this necessary? Is this something we should be addressing in a separate pull request, maybe?

Reply:
I created a separate [PR with test fixes](https://github.com/scrapy/scrapy/pull/5160)

Reply:
Without PR mentioned above tests for this(or any other) PR doesn't become green